src/FixerFactory.php

```diff
@@ -179,6 +179,8 @@ public function useRuleSet(RuleSetInterface $ruleSet)
                 } else {
                     throw new InvalidFixerConfigurationException($fixer->getName(), 'Is not configurable.');
                 }
+            } elseif ($fixer instanceof ConfigurableFixerInterface) {
+                $fixer->configure([]);
```

Comment:
this should not be needed

this means someone would also need to do this:
```php
$abc = new AbcFixer();
$abc->configure([]);
```
it should be enough to run constructor to have fixer ready for use